layout/example_test.go

```diff
@@ -127,6 +127,30 @@ func ExampleList() {
 	// 5
 }
 
+func ExampleContain() {
+	gtx := layout.Context{
+		Ops: new(op.Ops),
+		// Rigid constraints with both minimum and maximum set.
+		Constraints: layout.Constraints{
+			Max: image.Point{X: 100, Y: 100},
+		},
+	}
+
+	dims := layout.Contain.Layout(gtx, func(gtx layout.Context) layout.Dimensions {
```

Comment:
This is an example; can you describe what it intends to show?

Reply:
Yeah, this didn't make sense as an example, I converted it into a proper test.